addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -1184,13 +1189,29 @@
         // Color Pickers
         .o_we_so_color_palette.o_we_user_value_widget {
 
+            &:first-of-type(){
+                margin-right: $o-we-sidebar-content-field-multi-spacing * 4;
+            }
+
+            + .o_we_so_color_palette {
+                &:nth-of-type(2) {
+                    div {
```

Comment:
If you can add comments to explain what this targets, it's not really straightforward :grimacing: Also it is a bit scary to targets "all divs" maybe `> div` would be possible?